packages/jasmine/src/jasmine_runner.js

```diff
@@ -86,17 +80,6 @@ function main(args) {
                        // Filter out files from node_modules
                        .filter(f => !IS_NODE_MODULE.test(f))
 
```

Comment:
what about this feature where we only gathered coverage on production code rather than on tests? do we not need that anymore?

Reply:
ah yes, missed that in, added it back in the coverage processor